### ISSUE TITLE
libgtop2 dependency error with recent RHEL 7.6 release

### ISSUE THREAD

**Issue by jtrutwin** [mate-desktop/mate-desktop#337]
RHEL 7.6 was recently released and it appears they bumped the libgtop2 package to a newer version (2.38.0-3.el7).  When attempting to run yum update various MATE packages do not want to work with this version of libgtop2.  I also tried enabling epel-testing repo.  Note this is on a RHEL 7.6 "workstation" installation but I expect the problem is the same in RHEL-server/desktop.  Thank you.

#### Expected behaviour

No dependency errors :)

#### Actual behaviour

yum update fails with the following output:

# yum --enablerepo=epel-testing update

<snip>
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-system-monitor-1.16.0-1.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-applets-1.16.0-1.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: marco-1.16.1-3.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-disk-usage-analyzer-1.16.1-1.el7.x86_64
--> Finished Dependency Resolution
--> Running transaction check
---> Package kernel.x86_64 0:3.10.0-862.9.1.el7 will be erased
---> Package kernel-devel.x86_64 0:3.10.0-862.9.1.el7 will be erased
---> Package libgtop2.x86_64 0:2.34.2-2.el7 will be updated
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-system-monitor-1.16.0-1.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-applets-1.16.0-1.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: marco-1.16.1-3.el7.x86_64
--> Processing Dependency: libgtop-2.0.so.10()(64bit) for package: mate-disk-usage-analyzer-1.16.1-1.el7.x86_64
--> Finished Dependency Resolution
Error: Package: mate-applets-1.16.0-1.el7.x86_64 (@epel)
           Requires: libgtop-2.0.so.10()(64bit)
           Removing: libgtop2-2.34.2-2.el7.x86_64 (@anaconda/7.5)
               libgtop-2.0.so.10()(64bit)
           Updated By: libgtop2-2.38.0-3.el7.x86_64 (rhel-x86_64-workstation-7)
              ~libgtop-2.0.so.11()(64bit)
           Available: libgtop2-2.28.4-7.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.34.2-1.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.28.4-6.el7.i686 (rhel-x86_64-workstation-7)
               Not found
Error: Package: marco-1.16.1-3.el7.x86_64 (@epel)
           Requires: libgtop-2.0.so.10()(64bit)
           Removing: libgtop2-2.34.2-2.el7.x86_64 (@anaconda/7.5)
               libgtop-2.0.so.10()(64bit)
           Updated By: libgtop2-2.38.0-3.el7.x86_64 (rhel-x86_64-workstation-7)
              ~libgtop-2.0.so.11()(64bit)
           Available: libgtop2-2.28.4-7.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.34.2-1.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.28.4-6.el7.i686 (rhel-x86_64-workstation-7)
               Not found
Error: Package: mate-system-monitor-1.16.0-1.el7.x86_64 (@epel)
           Requires: libgtop-2.0.so.10()(64bit)
           Removing: libgtop2-2.34.2-2.el7.x86_64 (@anaconda/7.5)
               libgtop-2.0.so.10()(64bit)
           Updated By: libgtop2-2.38.0-3.el7.x86_64 (rhel-x86_64-workstation-7)
              ~libgtop-2.0.so.11()(64bit)
           Available: libgtop2-2.28.4-7.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.34.2-1.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.28.4-6.el7.i686 (rhel-x86_64-workstation-7)
               Not found
Error: Package: mate-disk-usage-analyzer-1.16.1-1.el7.x86_64 (@epel)
           Requires: libgtop-2.0.so.10()(64bit)
           Removing: libgtop2-2.34.2-2.el7.x86_64 (@anaconda/7.5)
               libgtop-2.0.so.10()(64bit)
           Updated By: libgtop2-2.38.0-3.el7.x86_64 (rhel-x86_64-workstation-7)
              ~libgtop-2.0.so.11()(64bit)
           Available: libgtop2-2.28.4-7.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.34.2-1.el7.x86_64 (rhel-x86_64-workstation-7)
               libgtop-2.0.so.10()(64bit)
           Available: libgtop2-2.28.4-6.el7.i686 (rhel-x86_64-workstation-7)
               Not found
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

#### Steps to reproduce the behaviour

Attempting to run yum update on a RHEL 7.5 system running MATE that will cause it to pull in the recently released 7.6 pacakges

#### MATE general version

mate-desktop-1.16.2-1.el7.x86_64

#### Package version

[root@vdlincstest01 view]# yum --enablerepo=epel-testing list libgtop2.x86_64
Installed Packages
libgtop2.x86_64                                                         2.34.2-2.el7                                                         @anaconda/7.5            
Available Packages
libgtop2.x86_64                                                         2.38.0-3.el7                                                         rhel-x86_64-workstation-7

#### Linux Distribution

RHEL 7.6

#### Link to downstream report of your Distribution

They won't do anything because MATE is "not supported"...  :/

**Comment by jtrutwin** [comment]
maybe this should be under mate-applets or mate-system-monitor?

**Comment by jtrutwin** [comment]
Workaround:

```bash
yum update --exclude=libgtop2 --exclude=gnome-shell
```

**Comment by scottsisco** [comment]
I am having the exact same issue.
